From: Marco Nenciarini <marco.nenciarini@devise.it>
Date: Tue, 2 Feb 2010 12:16:49 +0100
Subject: [PATCH] Fix whatis entries

Fix whatis entry of uuid.1, uuid.3ossp and uuid++.3ossp manpages
---
 uuid++.pod   |    2 +-
 uuid.pod     |    2 +-
 uuid_cli.pod |    2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/uuid++.pod b/uuid++.pod
index 8b5a4b1..89c5efb 100644
--- a/uuid++.pod
+++ b/uuid++.pod
@@ -31,7 +31,7 @@
 
 =head1 NAME
 
-B<OSSP uuid> - B<Universally Unique Identifier> (C++ API)
+B<uuid++> - B<Universally Unique Identifier> (C++ API)
 
 =head1 VERSION
 
diff --git a/uuid.pod b/uuid.pod
index 4ad3742..0179a46 100644
--- a/uuid.pod
+++ b/uuid.pod
@@ -31,7 +31,7 @@
 
 =head1 NAME
 
-B<OSSP uuid> - B<Universally Unique Identifier>
+B<uuid> - B<Universally Unique Identifier>
 
 =head1 VERSION
 
diff --git a/uuid_cli.pod b/uuid_cli.pod
index ddec6bb..df9dc83 100644
--- a/uuid_cli.pod
+++ b/uuid_cli.pod
@@ -31,7 +31,7 @@
 
 =head1 NAME
 
-B<OSSP uuid> - B<Universally Unique Identifier Command-Line Tool>
+B<uuid> - B<Universally Unique Identifier Command-Line Tool>
 
 =head1 VERSION
 
-- 
